notebooks/query_uploaded_database.ipynb

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
<p>Ok so if I understand correctly this is another way to create labeling tasks right? Would it make sense to combine this with the other notebook so we don't have diverging dataset creation methods?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>